.stickler.yml

```diff
@@ -0,0 +1,8 @@
+linters:
+    flake8:
+    eslint:
+        config: './.eslintrc.json'
+    csslint:
+fixers:
+    enable: true
```

Comment:
Configure this file, same as the one from [MICROVERSE's config](https://github.com/microverseinc/linters-config/blob/master/javascript/.stickler.yml) file.